oboe/hello-oboe/src/main/cpp/jni_bridge.cpp

```diff
@@ -53,6 +53,20 @@ Java_com_google_sample_oboe_hellooboe_PlaybackEngine_native_1setToneOn(JNIEnv *e
     engine->setToneOn(isToneOn);
 }
 
+JNIEXPORT void JNICALL
+Java_com_google_sample_oboe_hellooboe_PlaybackEngine_native_1setAudioApi(JNIEnv *env, jclass type,
```

Comment:
In a proper language you would do
```
Java_com_google_sample_oboe_hellooboe_PlaybackEngine_native_1setAudioApi(
        JNIEnv *env, jclass type,
        jlong engineHandle...) {
```

Reply:
Not sure what you mean. In C++ you can do `void method(param1, ...)` but explicit parameter types are preferable in this case.

Reply:
break line after the opening parenthesis and put each parameter in one line.